cupy/cuda/function.pyx

```diff
@@ -155,7 +155,12 @@ cdef _launch(intptr_t func, Py_ssize_t grid0, int grid1, int grid2,
 
 cdef class Function:
 
-    """CUDA kernel function."""
+    """Class in charge of the execution of a compiled CUDA kernel function.
+
+    This class is used to access the runtime attributes of kernels compiled
+    via the :class:`~RawKernel`. It is mean to be accessed through
+    :attr:`~RawKernel.function`. 
```

Comment:
This is not the primary goal of this class. I think we just don't need to mention this.

Reply:
ok!